web/client/utils/mapinfo/wfs.js

```diff
@@ -29,10 +29,11 @@ import assign from 'object-assign';
 const buildRequest = (layer, { map = {}, point, currentLocale, params, maxItems = 10 } = {}, infoFormat, viewer, featureInfo) => {
     if (point?.intersectedFeatures) {
         const { features = [] } = point?.intersectedFeatures?.find(({ id }) => id === layer.id) || {};
+        const isHTMLOutput = infoFormat === 'text/html';
```

Comment:
I should use `isRemote` as a flag, to identify the supported formats for remote request.